database/src/main/resources/db/migration/V1_23__Add_tx_history_tables_and_drop_views.sql

```diff
@@ -0,0 +1,44 @@
+DROP MATERIALIZED VIEW IF EXISTS block_cache_tx_history_day;
+DROP MATERIALIZED VIEW IF EXISTS block_cache_tx_history_hour;
+
+-- HOURLY count of Tx history
+CREATE TABLE IF NOT EXISTS block_cache_hourly_tx_counts
+(
+    block_timestamp TIMESTAMP NOT NULL,
```

Comment:
Add `id` on here, I know its essentially useless, but itll make the entities and queries much happier